webconnector/frontend/false/_cacache/index-v5/2e/66/d17574b72851a12c5b1cbd4c7b2c5d4205762cc3d80db67e6725bb515b20

```diff
@@ -0,0 +1,2 @@
+
+d7ec3af497f1eb23fd0a5296434bcd485d04ac7b	{"key":"pacote:range-manifest:https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.10.5.tgz:sha1-M2w+/BIgrc7dosn6tntaeVWjNlg=","integrity":"sha512-C2EkHXwXvLsbrucJTRS3xFHv7Mf/y9klmKDxPTE8yevCoH5h8Ae69Y+/lP+ahpW91crnzgO78elOk2E6APJfIQ==","time":1607423482007,"size":1,"metadata":{"id":"regenerator-runtime@0.10.5","manifest":{"name":"regenerator-runtime","version":"0.10.5","dependencies":{},"optionalDependencies":{},"devDependencies":{},"bundleDependencies":false,"peerDependencies":{},"deprecated":false,"_resolved":"https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.10.5.tgz","_integrity":"sha1-M2w+/BIgrc7dosn6tntaeVWjNlg=","_shasum":"336c3efc1220adcedda2c9fab67b5a7955a33658","_shrinkwrap":null,"bin":null,"_id":"regenerator-runtime@0.10.5"},"type":"finalized-manifest"}}
```

Comment:
this seems to be a new cache that was added? Why, and what does it do?
And in any case, it should not be in the repository, right? Could somebody clarify this?